beetmoverscript/test/test_zip.py

```diff
@@ -0,0 +1,418 @@
+import contextlib
```

Comment:
Needless to say this is simply beautiful!

Reply:
Thank you :)